salt/cloud/clouds/saltify.py

```diff
@@ -47,30 +54,150 @@ def __virtual__():
     return True
 
 
-def list_nodes():
+def _get_connection_info():
     '''
-    Because this module is not specific to any cloud providers, there will be
-    no nodes to list.
+    Return connection information for the passed VM data
     '''
-    return {}
+    vm_ = get_configured_provider()
+
+    try:
+        ret = {'username': vm_['username'],
+               'password': vm_['password'],
+               'eauth': vm_['eauth'],
+               'vm': vm_,
+               }
+    except IndexError:
+        raise SaltCloudException(
+            'Configuration must define salt-api "username", "password" and "eauth"')
+    return ret
 
 
-def list_nodes_full():
+def avail_locations(call=None):
     '''
-    Because this module is not specific to any cloud providers, there will be
-    no nodes to list.
+    This function returns a list of locations available.
+    salt-cloud --list-locations my-cloud-provider
+    [ saltify will always returns an empty dictionary ]
     '''
+
     return {}
 
 
-def list_nodes_select():
+def avail_images(call=None):
+    '''This function returns a list of images available for this cloud provider.
```

Comment:
I would love to put a CLI example with this one, but I have no idea how to call this function. There are no examples in any of the other drivers. Since I don't know how to use it, I can't test it.  Any body care to supply a working example?